setup.py

```diff
@@ -0,0 +1,50 @@
+from setuptools import setup
+
+def get_version():
+    version = {}
+    with open('bigquery_etl/version.py') as fp:
+        exec(fp.read(), version)
+
+    return version['__version__']
+
+
+setup(
+    name="mozilla-bigquery-etl",
+    version=get_version(),
+    use_incremental=True,
+    author="Mozilla Corporation",
+    author_email="fx-data-dev@mozilla.org",
+    description="Tooling for building derived datasets in BigQuery",
+    url="https://github.com/mozilla/bigquery-etl",
+    packages=["bigquery_etl", "bigquery_etl.cli", "bigquery_etl.metadata"],
+    package_data={},
+    install_requires=[
+        "gcloud",
+        "google-cloud-bigquery",
+        "google-cloud-storage",
+        "Jinja2",
+        "pytest-black",
+        "pytest-docstyle",
+        "pytest-flake8",
+        "pytest-mypy",
+        "pytest-xdist",
+        "pytest-dependency",
```

Comment:
`pytest-xdist` and `pytest-dependency` shouldn't be install dependencies. `python-xdist` provides options for running pytest on this repo (and tests aren't usually included in a python package), while `python-dependency` isn't used in this repo anymore, and should also be removed from `requirements.in`